helper/diffPlaces.js

```diff
@@ -40,6 +40,11 @@ function isLayerDifferent(item1, item2){
   return false;
 }
 
+function isUsState(item) {
```

Comment:
nit: I would do the later check first since it's cheap and quickly discards most undesirable cases

Reply:
it might be possible to use lodash `_.head` to make it even simpler